diff --git a/node_modules/adminjs/src/backend/decorators/property/property-decorator.ts b/node_modules/adminjs/src/backend/decorators/property/property-decorator.ts
index a09cdb1..c0e85ea 100644
--- a/node_modules/adminjs/src/backend/decorators/property/property-decorator.ts
+++ b/node_modules/adminjs/src/backend/decorators/property/property-decorator.ts
@@ -182,6 +182,7 @@ class PropertyDecorator {
    * @param {'list' | 'edit' | 'show' | 'filter'} where
    */
   isVisible(where: PropertyPlace): boolean {
+    // @ts-ignore
     if (typeof this.options.isVisible === 'object' && this.options.isVisible !== 'null') {
       return !!this.options.isVisible[where]
     }
diff --git a/node_modules/adminjs/src/frontend/components/app/action-header/styled-back-button.tsx b/node_modules/adminjs/src/frontend/components/app/action-header/styled-back-button.tsx
index bed93f3..a2d7279 100644
--- a/node_modules/adminjs/src/frontend/components/app/action-header/styled-back-button.tsx
+++ b/node_modules/adminjs/src/frontend/components/app/action-header/styled-back-button.tsx
@@ -10,6 +10,7 @@ import {
 import allowOverride from '../../../hoc/allow-override'
 
 // eslint-disable-next-line @typescript-eslint/no-unused-vars
+// @ts-ignore
 const StyledLink = styled(({ rounded, ...rest }) => <RouterLink {...rest} />)<ButtonProps>`${ButtonCSS}`
 
 export type StyledBackButtonProps = {
