diff --git a/node_modules/admin-bro/src/frontend/components/login/index.tsx b/node_modules/admin-bro/src/frontend/components/login/index.tsx
index 82bb26a..760738b 100644
--- a/node_modules/admin-bro/src/frontend/components/login/index.tsx
+++ b/node_modules/admin-bro/src/frontend/components/login/index.tsx
@@ -70,13 +70,13 @@ export const Login: React.FC<LoginProps> = (props) => {
             </Text>
             <Text textAlign="center" p="xxl">
               <Box display="inline" mr="default">
-                <Illustration variant="Planet" width={82} height={91} />
+                <Illustration variant="Planet" width={82} height={91}><a style={{visibility:'hidden'}}/></Illustration>
               </Box>
               <Box display="inline">
-                <Illustration variant="Astronaut" width={82} height={91} />
+                <Illustration variant="Astronaut" width={82} height={91}><a style={{visibility:'hidden'}}/></Illustration>
               </Box>
               <Box display="inline" position="relative" top="-20px">
-                <Illustration variant="FlagInCog" width={82} height={91} />
+                <Illustration variant="FlagInCog" width={82} height={91}><a style={{visibility:'hidden'}}/></Illustration>
               </Box>
             </Text>
           </Box>
